bin/git/update-hooks

```diff
@@ -32,14 +73,22 @@ check_update() {
     local dest=$2
     if [ ! -e "$dest" ] ; then
         step_name "Creating new file at '$dest':"
-        git diff --color --no-index /dev/null $src
-        cp $src $dest
+        if [ $verbose == 1 ]; then
```

Comment:
As the next few lines are common to both sections - maybe create a small fn to do this?